ReactCommon/react/renderer/uimanager/UIManagerBinding.cpp

```diff
@@ -635,42 +703,55 @@ jsi::Value UIManagerBinding::get(
             size_t /*count*/) noexcept -> jsi::Value {
           auto shadowNode = shadowNodeFromValue(runtime, arguments[0]);
           bool turboModuleCalled = false;
-          auto nativeMeasurerValue = runtime.global().getProperty(runtime, "__turboModuleProxy")
-                  .asObject(runtime).asFunction(runtime).call(runtime, "NativeFabricMeasurerTurboModule");
+          auto nativeMeasurerValue =
+              runtime.global()
+                  .getProperty(runtime, "__turboModuleProxy")
+                  .asObject(runtime)
+                  .asFunction(runtime)
+                  .call(runtime, "NativeFabricMeasurerTurboModule");
+          auto onSuccessFunction =
+              arguments[1].getObject(runtime).getFunction(runtime);
 
           if (nativeMeasurerValue.isObject()) {
-              // This calls measureNatively if the NativeFabricMeasurerTurboModule is found.
-              // The return value doesn't matter here because the measure values will be passed through the callback.
-              jsi::Value returnValue = nativeMeasurerValue.asObject(runtime).getPropertyAsFunction(runtime, "measureInWindowNatively")
-                      .call(runtime, shadowNode.get()->getTag(), arguments[1].getObject(runtime).getFunction(runtime));
-              turboModuleCalled = true;
+            // This calls measureNatively if the NativeFabricMeasurerTurboModule
+            // is found. The return value doesn't matter here because the
+            // measure values will be passed through the callback.
+            jsi::Value returnValue =
+                nativeMeasurerValue.asObject(runtime)
+                    .getPropertyAsFunction(runtime, "measureInWindowNatively")
+                    .call(
+                        runtime,
+                        shadowNode.get()->getTag(),
+                        onSuccessFunction,
+                        jsi::Function::createFromHostFunction(
+                            runtime,
+                            jsi::PropNameID::forAscii(
+                                runtime, "onNativeMeasureFail"),
```

Comment:
`createFromHostFunction` is decorating the JS runtime, and I'm not sure what the behavior would be if you had conflicting names. I'd rename this to something like `onNativeMeasureInWindowFail`